data/archiver/ingest_api.py

```diff
@@ -0,0 +1,56 @@
+import json
+import requests
+import functools
+import logging
+from data.archiver.config import INGEST_API
+
+def handle_exception(f):
+    @functools.wraps(f)
+    def func(*args, **kwargs):
+        try:
+            return f(*args, **kwargs)
+        except Exception as e:
+            #print(f'Ingest API exception in {f.__name__}')
```

Comment:
This looks like an exception that would not be logged anywhere. Is this intentional.